GDJS/Runtime/bitmapfontmanager.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
Why put in in GDJS? Many games probably won't use it and adding it without needing it is against the philosophy of extensions. Why don't you just add it as include of bitmap objects?

Reply:
Yeah agree, I was about to make that comment :) We can always move this later though @Bouh, for now you can concentrate on the rest, but we'll certainly want to make this manager part of the extension only.